changelog/16950.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
We don't need changelogs for changes that don't impact the binary.

Reply:
Oh, woops. I did a quick check to see if there were docs changelogs and there were some. Should I remove it, or leave it in?